.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ['1']  # Test against current minor release (at least 1.3)
```

```suggestion
        version: ['1.3', '1']  # Test against current minor release (at least 1.3)
```
This would allow to test against the minimal version (1.3) and latest stable